src/backend/web/routes/feeds.js

```diff
@@ -34,21 +35,42 @@ feeds.get('/:id', async (req, res) => {
 
   try {
     const feed = await Feed.byId(id);
-
-    // If the object we get back is empty, use 404
```

Comment:
Not sure why you removed these lines.  Not critical, but I'd avoid changing things not related to your code.

Reply:
Misclick :)